docs/installation.md

```diff
@@ -131,6 +131,18 @@ extraWhitelistPaths:
   - '/ext/faster'
 ```
 
+## Configure behind a load balancer
+
+A possible deployment of Trino Gateway is to run multiple instances of Trino 
+Gateway behind another generic load balancer, such as a load balancer from 
+your cloud hosting provider. In this deployment you must configure the 
+`serverConfig` to include enabling process forwarded HTTP headers:
+
+```yaml
+serverConfig:
+    http-server.process-forwarded: true
```

Comment:
nit: use two spaces for indent

Reply:
ok .. will adjust in other docs to that as well in a follow up